impeller/geometry/wangs_formula.cc

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/geometry/wangs_formula.h"
+
+namespace impeller {
+
+namespace {
+
+// Don't allow linearized segments to be off by more than 1/4th of a pixel from
+// the true curve.
+constexpr static Scalar kPrecision = 4;
+
+static inline Scalar length(Point n) {
+  Point nn = n * n;
+  return std::sqrt(nn.x + nn.y);
+}
+
+static inline Point Max(Point a, Point b) {
```

Comment:
Don't we already have `Point::Max` and `Point::GetLength`?

Reply:
Switched to Point::Max. The length measurement is different however.